docs/detections/detections-ui-exceptions.asciidoc

```diff
@@ -5,6 +5,7 @@
 To prevent the creation of unwanted alerts, you can add exceptions to the following rule types:
 
 * Custom query
+* Threshold
```

Comment:
I _believe_ that now makes it all rules accepting exceptions. Maybe we no longer need this snippet? @peluja1012 can you provide a sanity check here :)